v3/mqtt_dummy.go

```diff
@@ -38,9 +41,16 @@ var (
 	DefaultUsers    = []string{"good", "1234"}
 	DefaultSubData  []byte
 
+	DefaultItems = []*KeyValue{
+		&KeyValue{Key: "key1", Value: "value1", ModificationTime: time.Now()},
+		&KeyValue{Key: "key2", Value: "value2", ModificationTime: time.Now()},
+	}
+
 	currentDevice = "" // XXX - fix the logic that stores this globally
 
 	globalConf DummyConfig
+	packetID   uint32 = 1
```

Comment:
Why it this variable `uint32`? Packet id specification is `uint16`, I think.

Reply:
You're right, but atomic only has add for 32 and 64.